aws/rapids_ec2_mnmg.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.    taxi_df.head()
</pre><p>This line will take a lot of time to run if the dataset is bigger and will unnecessarily read the csv once and then won't persist it in the workers. Should we persist this before hand? You may also remove this line if not necessary. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Thanks, I added this for debugging!</p>